Amethyst/Layout/Layout.swift

```diff
@@ -115,12 +116,23 @@ class ReflowOperation: Operation {
     let screen: NSScreen
     let windows: [SIWindow]
     let frameAssigner: FrameAssigner
+    public var onReflowCompletion: (() -> Void)?
 
     init(screen: NSScreen, windows: [SIWindow], frameAssigner: FrameAssigner) {
         self.screen = screen
         self.windows = windows
         self.frameAssigner = frameAssigner
+        self.onReflowCompletion = nil
         super.init()
+        self.completionBlock = {
```

Comment:
The operation setting its own completion block feels a little strange. Something that creates a reflow operation would have to know the internals to know that it's not supposed to use the operation's `completionBlock` property. Is there a reason to not just let the screen manager set the completion block directly?

Reply:
I wanted to enforce `guard !self.isCancelled else { return }` in a central place, let me try it a different way.